test/JDBC/upgrade/13_6/schedule

```diff
@@ -334,5 +334,3 @@ BABEL-2999
 drop_index-before-15_6-or-16_2
 permission_restrictions_from_pg
 BABEL-730-before-15_6-or-16_1
-GRANT_SCHEMA
-babel-4517
```

Comment:
We are removing BABEL-4517 file from many schedule files. It seems unrelated to GRANT ON SCHEMA. Could you double check if that should be done?

Reply:
Done. It should be there.